packages/opentelemetry-sdk-trace-web/src/utils.ts

```diff
@@ -85,7 +85,12 @@ export function addSpanNetworkEvents(
   addSpanNetworkEvent(span, PTN.DOMAIN_LOOKUP_START, resource);
   addSpanNetworkEvent(span, PTN.DOMAIN_LOOKUP_END, resource);
   addSpanNetworkEvent(span, PTN.CONNECT_START, resource);
-  addSpanNetworkEvent(span, PTN.SECURE_CONNECTION_START, resource);
+  if (
+    hasKey(resource as PerformanceResourceTiming, 'name') &&
+    getProtocol((resource as PerformanceResourceTiming)['name']) === 'https:'
```

Comment:
Simpler:

```suggestion
    (resource as PerformanceResourceTiming)['name'].startsWith('https:')
```

(`resource.name` is already an absolute url string)